integration/environment/machine.go

```diff
@@ -212,7 +212,7 @@ func (m *Machine) PVCsDeleted(namespace string) (bool, error) {
 
 // WaitForStatefulSet blocks until all statefulset pods are running. It fails after the timeout.
 func (m *Machine) WaitForStatefulSet(namespace string, labels string) error {
-	return wait.PollImmediate(m.pollInterval, m.pollTimeout, func() (bool, error) {
+	return wait.PollImmediate(m.pollInterval, 500*time.Second, func() (bool, error) {
```

Comment:
Why not increase `m.pollTimeout`? Is this timeout special?

Reply:
Why wait 8 minutes? This is a bad idea.
We should redesign our tests if something like this is required.

Reply:
I was just testing this flaky test. I added a wip tag too.

Reply:
Ah, I got scared :P